tools/testing-gui/src/pages/THORNodePage.tsx

```
@@ -33,109 +33,121 @@ export default function THORNodePage() {
-        <p className="text-gray-500 dark:text-gray-400 mt-1">
-          Manage and query THORChain validator nodes
-        </p>
+    <div className="p-6">
+      <div className="max-w-2xl mx-auto">
+        <div className="space-y-6">
+          {/* Header */}
+          <div className="bg-white dark:bg-gray-800 rounded-lg shadow p-6">
+            <h2 className="text-2xl font-bold text-gray-900 dark:text-gray-100">THORNode Operations</h2>
+            <p className="text-gray-500 dark:text-gray-400 mt-1">
+              Manage and query THORChain validator nodes
+            </p>
```

ESLint

[error] Replace `⏎··············Manage·and·query·THORChain·validator·nodes⏎············` with `Manage·and·query·THORChain·validator·nodes`

(prettier/prettier)